src/app/tests/TestCommandInteraction.cpp

```diff
@@ -475,242 +475,6 @@ void TestCommandInteraction::AddInvokeResponseData(nlTestSuite * apSuite, void *
     }
 }
 
-void TestCommandInteraction::TestCommandSenderWithWrongState(nlTestSuite * apSuite, void * apContext)
```

Comment:
Adding blocking comment to check which tests actually need to be in order and move only these (to reduce the number of changed lines).